seagoat/engine.py

```diff
@@ -75,7 +75,7 @@ def __init__(self, path: str):
                 ripgrep.initialize(self.repository),
             ],
             "sync": [
-                chroma.initialize(self.repository),
+                chroma.initialize(self.repository, self.config["client"]["provider"]),
```

Comment:
actually, it might be possible to just get the config directly in the chroma source? see more in this thread: https://github.com/kantord/SeaGOAT/pull/292#discussion_r1347355122

Reply:
Perfect, I have made the changes to read the config directly in `chroma.py`